.github/workflows/ci_macos_mu4.yml

```diff
@@ -56,7 +56,10 @@ jobs:
     - name: "Configure workflow"
       env:
         pull_request_title: ${{ github.event.pull_request.title }}
-      run: |
+        SENTRY_SERVER_MU4_KEY: ${{ secrets.SENTRY_SERVER_MU4_KEY }}
+        SENTRY_SERVER_SANDBOX_KEY: ${{ secrets.SENTRY_SERVER_SANDBOX_KEY }}
+        SENTRY_PROJECT: ${{ github.event.inputs.sentry_project }}
+       run: |
```

Comment:
This change needs to be removed, it leads to an error (I fixed it for the master)

Reply:
Oops, sorry